effekt/jvm/src/main/scala/effekt/Repl.scala

```diff
@@ -215,21 +215,22 @@ class Repl(driver: Driver) extends REPL[Tree, EffektConfig, EffektError] {
     case d: Def =>
       val extendedDefs = module + d
       val decl = extendedDefs.make(UnitLit())
+      val output = config.output()
 
       runFrontend(source, decl, config) { cu =>
         module = extendedDefs
 
         // try to find the symbol for the def to print the type
-        (context.symbolOption(d.id) match {
+        d.ids.foreach(id => (context.symbolOption(id) match {
```

Comment:
I believe this is not the correct implementation. It should print
```
x:Int, y:String, z:Bool
```
not
```
x: Int
y: String
z: Bool
```

or what do you think?

Reply:
No, this is the correct behavior in this case. It can still be refactored, though.